src/components/InterestForm/styles.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const FormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 60%;
+  float: right;
+  margin-top: 1rem;
+  gap: 0.5rem;
+`;
+
+export const FormTextArea = styled.textarea`
+  height: 16vh;
```

Comment:
Very dangerous to use `vh` here, I would suggest using `rem` with `min-height` instead. `vh` may seem fine on a desktop view, but once you're in the land of the mobile devices, viewport units get weird.